website/docs/command-serve.md

```diff
@@ -37,18 +39,108 @@ import TabItem from '@theme/TabItem';
 
 ### Usage
 
+The bare minimum you need is a GraphQL file with your data models. Create a file called `Note.graphql` and add the following:
+
+```graphql
+""" @model """
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  likes: Int
+}
+```
+
+The `@model` annotation indicates that `Note` is a data model and Graphback will generate resolvers, a CRUD service and data source for it. You can learn how to build more complex data models in [Data Model](https://graphback.dev/docs/model/datamodel#model).
+
+#### Running your codeless GraphQL server
+
+To start your server, run the following command from the same directory as `Note.graphql`:
+
+```bash
+graphql serve Note.graphql
+```
+
+This will start a GraphQL server on a random port using the `Note.graphql` data models we just added.
+
+You can customise the directory of the data models:
+
+```bash
+graphql serve ./path/to/models
+```
+
+You can also specify where to load the data models from with a Glob pattern:
+
+```bash
+graphql serve ./schema/**/*.graphql
+```
+
+You can specify which port to start the server on:
+
+```bash
+$ graphql serve ./path/to/models --port 8080
+
+Starting server...
+
+Listening at: http://localhost:8080/graphql
+```
+
+### Enable Data Synchronization
+
+GraphQL Serve can also operate on data sync models. Under the hood this uses the [Data Sync](https://graphback.dev/docs/datasync/intro) package. 
+To enable data synchronization, all we need to do is enable datasync capabilities on our models via the `@datasync` annotation.
+
+For the `Note` model defined above, this would look like: 
+
+```graphql
+""" 
+@model
+@datasync 
+"""
+type Note {
+  id: ID!
```

Comment:
Data sync will fail unless you use `_id: GraphbackObjectID` since this is MongoDB.

```suggestion
  _id: GraphbackObjectID!
```

Reply:
Good catch!